.shippable.yml

```diff
@@ -18,4 +18,4 @@ install:
   - "firefox -v || echo 'No firefox installed.'"
   - "shippable_retry ./setup.sh install"
 script:
-  - "grunt test"
+  - "grunt test --firefox-bin /usr/bin/firefox"
```

Comment:
Yeah exactly - jpm currently uses hardcoded default paths for Firefox that are actually incorrect on Debian/Ubuntu systems. See: https://github.com/canuckistani/fx-runner/pull/1